contracts/BSCValidatorSet.sol

```diff
@@ -1122,6 +1121,12 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     address validator;
     bool isFelony;
 
+    for (uint i; i<_validatorSet.length; ++i) {
```

Comment:
```
      for (uint k; k<_validatorSet.length; ++k) {
        if (_validatorSet[k].consensusAddress == validator) {
          _validatorSet[k].jailed = true;
          ++numOfFelony;
          break;
        }
      }
```
because of above logic, it is possible that one `_validatorSet[i]` cause `numOfFelony` increase twice.

Reply:
Fixed